README.md

```diff
@@ -53,18 +53,39 @@ Now that we've covered the basics of using `dotnew new`, lets move on to info fo
 If you're authoring templates, or interested in contributing to this repo, then you're likely interested in how to use the latest version of this experience.
 The steps required are outlined below.
 
-Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+## Prerequisites
+In order to work with this project, you will need the following.
+- [Visual Studio 2017](https://www.visualstudio.com/)
 
-Step 2: Clone this repository to your local machine
+## Setup
 
-Step 3:  Run the setup script
- - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
- - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
+Step 1:
+- Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli).
+- Typically, development is  done against the latest RC FTS Release.  
+  - It is best to ask the team to clear up any compatibility concerns before starting.
 
-When running the setup script, the existing built in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
+Step 2:
+- Clone this repository to your local machine.
+
+```bash
+git clone https://github.com/dotnet/templating.git
+```
+
+- There is no guarantee that **master** is the current working branch. 
+  - It is recommended that you check with the team to ensure you are developing or building from the correct branch. 
+
+Step 3: 
+- Open up a command prompt and navigation to the root of your source code.
+- Run the setup script.
+     - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
+     - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
+- When running the setup script, the existing built-in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
 files with the latest Template Engine.
+- Closing your console window or "session" will destroy your current instance of `dotnet new3`.
+  - The next time a console window is opened, you will again need to run the setup script. 
```

Comment:
This is currently true for Windows (until #401 gets merged). It's not currently true for non-Windows.